charts/graphprotocol-indexer-agent/templates/deployment.yaml

```diff
@@ -62,7 +62,14 @@ spec:
             - name: INDEXER_AGENT_GRAPH_NODE_ADMIN_ENDPOINT
               value: {{ required "config.graphEndpoints.admin wasn't specified" .Values.config.graphEndpoints.admin | quote }}
             - name: INDEXER_AGENT_GRAPH_NODE_STATUS_ENDPOINT
-              value: {{ required "config.graphEndpoints.status wasn't specified" .Values.config.graphEndpoints.status | quote }}            
+              value: {{ required "config.graphEndpoints.status wasn't specified" .Values.config.graphEndpoints.status | quote }}
+            {{- if hasKey .Values.postgres.password "fromSecret" }}
+            - name: INDEXER_AGENT_POSTGRES_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.postgres.password.fromSecret.name }}
```

Comment:
Just a question: I generally ok with the proposed approach to make it possible to read password from some external secret. But can we simplify and just get rid of the `fromSecret` option? Maybe having a bit of redundancy for the sake of simplicity is not so bad idea.